server/statshouse/statshouse-manager.cpp

```diff
@@ -410,13 +410,15 @@ void StatsHouseManager::add_slow_net_event_stats(const slow_net_event_stats::sta
                             break;
                           }
 
-                          static constexpr size_t CURL_URL_MAX_LEN = 100;
-                          std::string_view curl_url = curl_response_stat.opt_url.value_or(std::string_view{"unknown"});
+                          static constexpr size_t DEFAULT_CURL_URL_MAX_LEN = 100;
```

Comment:
I'd suggest following:
```
static constexpr size_t CURL_URL_MAX_LEN = 100;
std::string_view curl_url = curl_response_stat.opt_url.value_or(std::string_view{"unknown"});
if (auto pos = curl_url.find_first_of('?'); pos != std::string_view::npos) {
  curl_url.remove_suffix(curl_url.size() - pos);
}

client.metric("kphp_slow_curl_response")
  tag(curl_kind)
  .tag(std::string_view{curl_url.data(), std::min(curl_url.size(), CURL_URL_MAX_LEN)})
  .write_value(curl_response_stat.response_time);
 
```

Here we use more performant overload of `find_first_of` and the code overall is more clear imo